corehq/motech/repeaters/management/commands/populate_repeater_names.py

```diff
@@ -16,13 +17,17 @@ def handle(self, *args, **options):
         self._total_repeaters_to_update = Repeater.objects.filter(name=None).count()
         while len(repeaters) > 0:
             self._show_progress(repeater_count=len(repeaters))
+            connections = {
+                cx.id: cx for cx in ConnectionSettings.objects.filter(
+                    id__in=[r.connection_settings_id for r in repeaters])
+            }
             for repeater in repeaters:
-                repeater.name = repeater.connection_settings.name
+                repeater.name = connections[repeater.connection_settings_id].name
                 repeater.save()
             repeaters = self._get_repeaters_with_empty_names()
 
     def _get_repeaters_with_empty_names(self):
-        return Repeater.objects.filter(name=None).select_related('connection_settings')[:CHUNKSIZE]
+        return Repeater.objects.filter(name=None)[:CHUNKSIZE]
```

Comment:
... alternatively, we could just delete this management command. iirc it was to backfill repeater names.

Reply:
@SmittieC would you like to delete it (in a separate PR)? Seems like you might know best if it's no longer needed.

Reply:
Sure I don't mind. It was just a temp management command

Reply:
Actually, it seems like this is used by the `00012_create_default_names_for_repeaters` migration. I can just move this code to that migration file, then we should be able to remove this command.